conf/test.config

```diff
@@ -24,8 +24,8 @@ params {
     // TODO nf-core: Give any required params for the test so that command line flags are not needed
     input                                 = 'https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/samplesheet.csv'
     databases                             = 'https://raw.githubusercontent.com/nf-core/test-datasets/taxprofiler/database.csv'
-    perform_shortread_clipmerge           = true
-    perform_longread_clip                 = false
+    perform_shortread_qc           = true
+    perform_longread_qc                 = false
```

Comment:
Is it `false` here because it is an optional step?

Reply:
Copy paste error :sweat_smile: